lib/src/pooling/normal/default_world_pool.dart

```diff
@@ -296,9 +272,9 @@ class DefaultWorldPool implements IWorldPool {
 
   List<Vector2> getVec2Array(int argLength) {
     if (!_avecs.containsKey(argLength)) {
-      List<Vector2> ray = new List<Vector2>(argLength);
+      List<Vector2> ray = List<Vector2>(argLength);
```

Comment:
Suggestion: List.generate

Reply:
List.filled is even nicer when you don't need to know the index :)